src/clusterfuzz/_internal/google_cloud_utils/batch.py

```diff
@@ -275,8 +275,8 @@ def _get_config_names(
     # If we are running in the oss-fuzz context, the project-specific config
     # is more specific and overrides the job-level one.
     if environment.get_value('PROJECT_NAME') == 'oss-fuzz':
-      project_name = job.project
-      oss_fuzz_project = data_types.OssFuzzProject.get_by_id(project_name)
+      oss_fuzz_project = data_types.OssFuzzProject.query(
+          data_types.OssFuzzProject.name == job.project).get()
```

Comment:
Why are we changing this?

Reply:
the entity `OssFuzzProject` doesn't have an `id`, and I couldn't find any other usage of `get_by_id` on the codebase, then I'm not sure with `get_by_id` it would work.